lib/Conversion/HWToSystemC/HWToSystemC.cpp

```diff
@@ -79,10 +79,29 @@ struct ConvertHWModule : public OpConversionPattern<HWModuleOp> {
         module.getLoc(), portDirections,
         ArrayAttr::get(rewriter.getContext(), portNames));
 
-    // Inline the HW module body into the SystemC module body.
-    Region &scModuleBodyRegion = scModule.getBodyRegion();
-    rewriter.inlineRegionBefore(module.getBodyRegion(), scModuleBodyRegion,
-                                scModuleBodyRegion.end());
+    // Create a systemc.ctor operation inside the module.
+    Block *moduleBlock = new Block;
```

Comment:
I was surprised to actually see a `new`. However, this appears to be the correct way to do this.